src/common/logger.ts

```diff
@@ -15,11 +20,8 @@ export const contextStorage = new AsyncLocalStorage<
 >()
 
 const setContext = format((info, _) => {
-  const context = contextStorage.getStore()
-  if (context) {
-    info.requestId = context!.get('requestId')
-    // info.jobId = context!.get('jobId')
-  }
+  const context = contextStorage.getStore() || new Map()
+  info.requestId = context.get('requestId')
```

Comment:
https://github.com/thematters/matters-server/pull/3636/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80 

IP and UA?

Reply:
here https://github.com/thematters/matters-server/pull/3636/files#diff-d6a621e5b8adb30e9f5b065e7d0c84185b704acb42df7009aa79d1fb397ecc40R88-R89